tutorial_devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,52 @@
+
+It also contains a wide range of pre-compiled binaries for multiple OS, including Windows, Linux and OSX, a variety of compilers, compiler versions, static and shared libraries, etc. These precompiled packages can be very useful to do quick experiments, testing, onboarding, and they also serve as a proof that the recipes for those packages can work solidly for that wide range of configurations.
+
+However, it is not a good practice to depend directly on ConanCenter for production systems, for several reasons:
+
+- ConanCenter continuously gets contributions, fixes and improvements from the community, but that also means that ConanCenter can only guarantee that it works with the latest Conan version (see the :ref:`stability commitment in the introduction<stability>`). If you want to update your Conan version at a different pace, ConanCenter packages that introduce latest version features will break.
+- The changes in recipes might also unpexectedly cause bugs and other issues. Recipes and package updates might bump dependencies versions to use the latest, forcing a continuous update over all dependencies versions. That can cause version conflicts if your project doesn't bump dependencies versions at the same pace.
+- Fixes to ConanCenter recipes can take a while to be merged (weeks), because there are too many contributions to be reviewed. If you rely on a fix that you contributed and it is urgent for your production system, you might need to wait.
+- Binary configurations can be discontinued in ConanCenter, when new compiler versions are released and ConanCenter starts to build them, it can drop some of the older versions.
+- Outages, maintenance windows and other service interruptions can happen, ranging from a few minutes to days. The infrastructure to host central repositories is typically large and complex, and relies on cloud providers that sometimes have their own issues, networks, CDN, etc.
+- Security issues. Even if the process to add packages to ConanCenter is very strict, with human reviews for every package and many automated checks and processes, eventually all systems are subject to attempts of supply chain attacks.
+
+Most of these reasons are not unique to Conan, but to every other package manager for other programming languages. The general devops known good practice is that you shouldn't rely on central repositories on production systems, but host your own copy of the packages that you need for production.
+
+This is the main idea: **host your own copy of the packages you need for production in your own server**. This could be done by just downloading binaries from ConanCenter and then uploading them to your own server. However, it is much better to fully own the complete supply chain and create the binaries in your own CI systems. So the recommended flow to use ConanCenter packages in production would be:
```

Comment:
I think that we could make the main idea sentence a subtitle? Maybe that way the communication of the idea is more powerful and users can get to the part where the practical solution is explained quicker.

Reply:
How a subtitle? An "Important" note just below the page main title?